src/main/java/com/vimeojam/Shorteo/model/TestRedis.java

```diff
@@ -18,4 +18,20 @@
 
     @NotNull
     private String shorteo_url;
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public String getShorteo_url() {
+        return shorteo_url;
+    }
+
+    public void setShorteo_url(String shorteo_url) {
+        this.shorteo_url = shorteo_url;
+    }
```

Comment:
'@Data' annotation provides all the getter and setter functions.
https://projectlombok.org/features/Data